src/main/java/org/betonquest/betonquest/quest/condition/weather/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * {@link org.betonquest.betonquest.api.quest.condition.PlayerCondition PlayerCondition}
```

Comment:
This is not a player condition. I also like the formulation "Conditions for weather" more.